### DISCUSSION TITLE
Tags are too close to one another on smartphone / simple theme

### DISCUSSION THREAD

**Discussion by hubyhuby** [GetPublii/Publii#1357] [Help] [answered]
![image](https://github.com/GetPublii/Publii/assets/26340942/1500f40d-60f8-46de-966a-f26539d3d454)

![image](https://github.com/GetPublii/Publii/assets/26340942/e47a305b-33e0-4b51-b5ba-8df805cfe4e7)

**Comment by bobmitro** [comment]
Any Url?

**Comment by hubyhuby** [comment]
yes, there    compass-financement  com
In the posts you sse tags

**Reply by bobmitro** [reply, marked as answer]
Add the following CSS rule using the Custom CSS tool:

```
.post__tag > li {
  display: inline-flex;
  margin-bottom: .2833333333rem;
}
```

**Reply by hubyhuby** [reply]
Thanks @bobmitro 
It s exactely what I needed!
This comunity is very helpfull and reactive <3
